lib/web/assets.go

```diff
@@ -21,53 +21,11 @@ import (
 	"io"
 	"net/http"
 	"os"
-	"path"
-	"path/filepath"
 	"strings"
 
 	"github.com/gravitational/trace"
 )
 
-// NewDebugFileSystem returns the HTTP file system implementation rooted
-// at the specified assetsPath.
-func NewDebugFileSystem(assetsPath string) (http.FileSystem, error) {
-	assetsToCheck := []string{"index.html", "/app"}
-	if assetsPath == "" {
-		exePath, err := executableFolder()
-		if err != nil {
-			return nil, trace.Wrap(err)
-		}
-
-		_, err = os.Stat(path.Join(exePath, "../../e"))
-		isEnterprise := !os.IsNotExist(err)
-
-		if isEnterprise {
```

Comment:
Does dropping this logic cause problems for e/ tests?

Reply:
It doesn't, we don't actually query anything other than the oss `index.html` and `config.js` files in the tests. That was added a couple years ago so I'm guessing that things have changed since then.